storage/store.go

```diff
@@ -500,11 +520,24 @@ func (s *Store) maybeSplitRangesByConfigs(configMap PrefixConfigMap) {
 	}
 }
 
+// ForceReplicationScan iterates over all ranges and enqueues any that need to be
+// replicated.
+func (s *Store) ForceReplicationScan() {
```

Comment:
Let's not export this method and make a note that it's for use from unittests only. Did you have some other use case in mind?